create CardsKreatywny, add json and types for Cards

Basilika-draconum opened the pull request:
create and add content in kreatywny-cards.json, created component card and container for cards for all dimensions, add type for 1 card

Basilika-draconum committed:
create CardsKreatywny, add json and types for Cards

create and add content in kreatywny-cards.json, created component card and container for cards for all dimensions, add type for 1 card

opanchen commented on src/data/kreatywny-cards.json:
файл логічніше назвати 'section-about' чи якось так (потім сюди додамо ще текст заголовка і тд), і вже в обʼєкті секції додати поле 'cards' з масивом даних, а в компоненті після імпорту витягнути те, що конкретно потрібно

Basilika-draconum replied:
виправила, одразу додала заголовок і підзаголовок

opanchen commented on src/components/ui/CardKreatywny/CardKreatywny.tsx:
```diff
@@ -0,0 +1,25 @@
+
+const CardKreatywny: React.FC<CardKreatywnyProps> = ({ card }) => {
+  const { id, title, description } = card;
+  const titleColorClass =
```
варто уникати подвійних/потрійних тернарників - це важко читається
треба замінити  або спростити тернарник до 1 рівня (якщо зручно, можна встановити і використовувати пакетик classNames, наприклад)

Basilika-draconum replied:
+

opanchen commented on src/components/ui/CardKreatywny/CardKreatywny.tsx:
```diff
@@ -0,0 +1,25 @@
+  );
+};
+
+export default CardKreatywny;
```
для компонентів використовуємо іменований експорт, заміни тут і в СаrdsKreatywny.tsx

Basilika-draconum replied:
+

opanchen commented on src/components/ui/CardKreatywny/CardKreatywny.tsx:
```diff
@@ -0,0 +1,25 @@
+        ? 'text-color-accent-primary'
+        : 'text-color-accent-extra';
+  return (
+    <li className="mx-auto flex h-[298px] w-[320px] flex-col justify-between rounded-3xl bg-white p-[23px] md:h-[319px] md:w-[596px] md:p-12">
```
висота картки на мобілці 253 px (https://prnt.sc/fiItSO0DBkD4)

Basilika-draconum replied:
+

Basilika-draconum committed:
rename json and fix reexport

rename json on section-about and add title and subtitle, edit components exports, edit condition for title in Card

Basilika-draconum committed:
solve conflicts
